extensions/scheduler/deployment/src/test/java/io/quarkus/scheduler/test/PausedSchedulerTest.java

```diff
@@ -33,15 +34,19 @@
     public void testSchedulerPauseMethod() throws InterruptedException {
         scheduler.pause();
         assertFalse(scheduler.isRunning());
+        Jobs.IS_WATCHING.set(true);
         assertFalse(Jobs.LATCH.await(3, TimeUnit.SECONDS));
     }
 
     static class Jobs {
+        static final AtomicBoolean IS_WATCHING = new AtomicBoolean(false);
         static final CountDownLatch LATCH = new CountDownLatch(2);
 
         @Scheduled(every = "1s")
         void countDownSecond() {
-            LATCH.countDown();
+            if (IS_WATCHING.get()) {
+                LATCH.countDown();
+            }
```

Comment:
This change is unrelated to my changes, but it was randomly failing for me. Using the is watching boolean, made the test more consistent.

Reply:
I've never seen this error before. Do you mind putting this in a separate commit / PR?

Reply:
I've deleted my changes here and let's see if the CI complains. If so, I will create a separate PR for this. 
Thanks!